NAMESPACE

```diff
@@ -20,4 +20,5 @@ export(raise_issue)
 export(save_rds)
 export(year_sector_data)
 export(year_sector_table)
+import(dplyr)
```

Comment:
are you sure you want to import dplyr here? It is more usual to import it in DESCRIPTION and then call dplyr functions throughtout with `dplyr::group_by()` etc.